redash/handlers/pdf.py

```diff
@@ -0,0 +1,28 @@
+import tempfile
+import os
+
+from flask import request, send_file
+from redash import models
+from redash.permissions import require_access, view_only, require_permission
+from redash.serializers import serialize_dashboard
+from redash.handlers.base import BaseResource, get_object_or_404
+
+class DashboardPdfDownload(BaseResource):
+    def get(self, dashboard_slug=None):
+        "dashboard = get_object_or_404(models.Dashboard.get_by_slug_and_org, dashboard_slug, self.current_org)"
+
+        self.record_event({
+            'action': 'download_pdf',
+            'object_type': 'dashboard',
```

Comment:
I think, it's useful to record also dashboard id here

Reply:
Will do.